validate/src/command.rs

```diff
@@ -250,22 +279,70 @@ pub fn name(value: impl AsRef<str>) -> Result<(), CommandValidationError> {
         Ok(())
     } else {
         Err(CommandValidationError {
-            kind: CommandValidationErrorType::NameInvalid,
+            kind: CommandValidationErrorType::NameLengthInvalid,
         })
     }
 }
 
+/// Validate the name of a [`ChatInput`] command or a [`CommandOption`] name.
+///
+/// The length of the name must be more than [`NAME_LENGTH_MIN`] and less than
```

Comment:
since we already check the length of the names in other places, i'd like to make this function only check the lower-caseness of a name. this also means we don't have to mix constants (even if they are the same length)

Reply:
this still needs addressing